public/static/app/perf/perf.js

```diff
@@ -329,7 +329,10 @@ $http.get(templateUrl).success(function(template) {
   }
 
   $scope.comparisonPct = function (compareSample, testName) {
-    const threadLevel = _.max($scope.selectedThreads[testName]);
+    let threadLevel = _.max($scope.selectedThreads[testName]);
```

Comment:
I find the fact that we are depending on [_.max](https://underscorejs.org/#max) returning __-Infinity__ for non numeric data totally non-obvious, especially since an empty array also returns __-Infinity__.

Can we add a const named THREAD_LEVEL_MAX (or something descriptive):
```
// _.max / https://underscorejs.org/#max returns -Infinity for non-numeric data. 
const THREAD_LEVEL_MAX = -Infinity
```

And use this for the checks.

```
   if(threadLevel === THREAD_LEVEL_MAX)
```

[suggestion] Add a _console.warn_ when __$scope.selectedThreads[testName]__ is empty.

Reply:
I didn't think of it that way, but I changed it to be more explicit about checking for the selection